### ISSUE TITLE
COntact & About combine

### ISSUE THREAD

**Issue by LaserRayGunBOSS** [thefootlightbar/thefootlightbar.github.io#9]
Can we delete the contact tab in the menu and just add that info in the about section? 

We want to add a new menu selection for film location info I am writing the copy now and we'll submit it as separate issue.

**Comment by meitar** [comment]
> Can we delete the contact tab in the menu and just add that info in the about section?

Yes, but these are two separate pages. You probably also want to combine the page contents, otherwise some of the information on your website will be inaccessible?

**Comment by LaserRayGunBOSS** [comment]
Yes, I would like to just add the contact info into the about section. Please

**Comment by meitar** [comment]
> Yes, I would like to just add the contact info into the about section.

Sounds good.

I'm going to wait on the copy for the new page so that I can make the change in a single commit; that way there will always be exactly seven items in the navigation bar and the layout will not be dramatically affected.
